contracts/CustomCrowdsale.sol

```diff
@@ -146,8 +146,8 @@ contract CustomCrowdsale is Ownable {
     }
 
     // Total Tokens held in the CustomCrowdsale
-    function getTokenTotalSupply() public view returns(uint) {
-        return token.totalSupply();
+    function getTokenTotalSupply(address src) public view returns(uint) {
+        return token.getBalance(src);
```

Comment:
Why that src? If you want to get the balance for the token just call the token. I thought this method was used to check the tokens of the `CustomCrowdsale` contract